packages/forms/src/fields/ObjectField.js

```diff
@@ -103,6 +103,9 @@ class ObjectField extends Component {
 		const { widget, ...options } = getUiOptions(uiSchema);
 		// widget
 		if (typeof widget === 'string') {
+			if (widget === 'hidden') {
+				return (<input type='hidden'/>);
```

Comment:
My proposal was if `ui:widget` set to hidden for Object type, then it should render nothing, wha do you think?